### DISCUSSION TITLE
Telescope branch sessions

### DISCUSSION THREAD

**Discussion by aserowy** [olimorris/persisted.nvim#68] [Q&A] [answered]
Heho,

first things first: tyvm for this awesome plugin!

Is there a way to configure the picker to show only cwd specific sessions regarding branches?

**Comment by olimorris** [comment, marked as answer]
Thanks for your kind words and glad you like it.

There's current no way to do that. I'd be happy to accept any PR that could add this, perhaps a user configurable option?
